.circleci/slack-notification.sh

```diff
@@ -17,9 +17,13 @@ if [ "${CIRCLE_BRANCH}" == "master" ]; then
     author=${author/valentin-pinkau/<@valentin>}
     channel="webknossos-bots"
     commitmsg=$(git log --format=%s -n 1)
+    pullregex="(.*)#([0-9]+)(.*)"
+    while [[ $commitmsg =~ $pullregex ]]
```

Comment:
Why do we have the `while`-loop? I would expect an `if`. Nothing important, just curious ;)

Reply:
I want to replace all occurences of the pattern with the respective linkified version. There are probably nicer ways to do this, but this does the trick. `if` would only run at most one replacement.